server.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import userRoutes from './routes/registerRoutes.js';
+import registerRoutes from './routes/registerRoutes.js';
+import protectedRoutes from "./routes/protectedRoutes.js";
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.json());
+app.use('/api', userRoutes);
```

Comment:
Good practice is to use /api/v1 as initial point, because with api evolution you might be wanting to introduce v2 for the same routes and keep v1 for backward compatibility